src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/Stream.WriteTests.cs

```diff
@@ -9,34 +9,56 @@
 
 namespace System.Text.Json.Serialization.Tests
 {
-    public static partial class StreamTests
+    public class StreamTests_Async : StreamTests
     {
+        public StreamTests_Async() : base(SerializationWrapperForStream.AsyncStreamSerializer) { }
+    }
+
+    public class StreamTests_Sync : StreamTests
+    {
+        public StreamTests_Sync() : base(SerializationWrapperForStream.SyncStreamSerializer) { }
+    }
+
+    public abstract partial class StreamTests
+    {
+        private SerializationWrapperForStream Serializer { get; }
```

Comment:
nit: per https://github.com/dotnet/runtime/pull/54632/files#r657472787
```suggestion
        private SerializationWrapperForStream JsonSerializerWrapper { get; }
```

Reply:
IMO "Serializer" is fine since the usages of it are either:
`Serializer.DeserializeWrapper(`
or
`Serializer.SerializeWrapper(`

Otherwise it gets a little verbose (`JsonSerializerWrapper.DeserializeWrapper`).

Also I believe based on the original naming discussion a long time ago, we didn't want to have "JsonSerializer" in the name to prevent potential search collisions with the real "S.T.J.JsonSerializer".

However, I'll add an Xml comment to the `Serializer` variable to make it more obvious.